Clinical-Genomics/scout
# Keep filter panel expanded when filters have been used by user

Fix #1944

This PR makes 
- variants, 
- SV-variants, 
- cancer-variants and 
- SV-cancer-variants 

pages more user-friendly by keeping the variants filtering panel open when results of filtering are returned to the user.

**How to test**:
1. Install the branch and try to filter variants of the types listed above using the filters panel. When filtered variants are returned, filters panel should stay open (POST requests). The panel should be collapsed when the user first goes to the variants pages or when the button "reset filters" is pushed.

1. Make sure that the filter panels stays also open when filtering SNV cancer variants with wrong params (filter form not validated and and an error is returned). 

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by ML
- [x] tests executed by ML, CR
